packages/hawtio/src/plugins/springboot/types.ts

```diff
@@ -37,3 +37,65 @@ export type Logger = {
   configuredLevel: string
   effectiveLevel: string
 }
+
+export class Trace {
+  timestamp: string
+  method: string = ''
+  path: string = ''
+  httpStatusCode: number = -1
+  timeTaken: string = '-1'
+  info: string = ''
+
+  constructor(traceData: unknown) {
+    const trace = traceData as JmXTrace
```

Comment:
receiving it as `unknown` and casting it to `JmxTrace` here is not a good api. rather, let's cast it before passing it to this constructer and explicitly declare what type it accepts.